diagnostic.md

```diff
@@ -7,40 +7,51 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+to work on this repo locally, we clone it onto our desired directory by using the
+command: git clone git@github.com:TronB/git-diagnostic.git
+
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+type in the command: git status, to see if there are any files that have been
```

Comment:
Before doing this, you would want to create a new branch to make certain that you are not pushing anything to master